src/main/java/io/spring/githubchangeloggenerator/ChangelogGenerator.java

```diff
@@ -16,6 +16,18 @@
 
 package io.spring.githubchangeloggenerator;
 
+import static io.spring.githubchangeloggenerator.ApplicationProperties.FormatPlaceholder.NUMBER;
+import static io.spring.githubchangeloggenerator.ApplicationProperties.FormatPlaceholder.TITLE;
+import static io.spring.githubchangeloggenerator.ApplicationProperties.FormatPlaceholder.URL;
+
+import io.spring.githubchangeloggenerator.ApplicationProperties.ExternalLink;
+import io.spring.githubchangeloggenerator.ApplicationProperties.IssueSort;
+import io.spring.githubchangeloggenerator.ApplicationProperties.PortedIssue;
+import io.spring.githubchangeloggenerator.github.payload.Issue;
+import io.spring.githubchangeloggenerator.github.payload.Label;
+import io.spring.githubchangeloggenerator.github.payload.User;
+import io.spring.githubchangeloggenerator.github.service.GitHubService;
+import io.spring.githubchangeloggenerator.github.service.Repository;
```

Comment:
Please correct the imports as per the problems reported by Checkstyle when you run `./gradlew build`.